rcl/include/rcl/event.h

```diff
@@ -30,13 +30,15 @@ extern "C"
 typedef enum rcl_publisher_event_type_t
 {
   RCL_PUBLISHER_OFFERED_DEADLINE_MISSED,
-  RCL_PUBLISHER_LIVELINESS_LOST
+  RCL_PUBLISHER_LIVELINESS_LOST,
+  RCL_PUBLISHER_OFFERED_INCOMPATIBLE_QOS
```

Comment:
nit: Add a trailing comma (that avoid this line to be modified in future modifications).

Reply:
Done.